specs/configure_monitoring.adoc

```diff
@@ -240,35 +240,27 @@ will look like:
 
 ----
 namespace.tendrl.node_monitoring:
-  Tendrl_definition:
-    flows:
-      ConfigureCollectd:
-        atoms:
-          - tendrl.node_agent.objects.Node.atoms.cmd
-        description: "Execute given command on given node"
-        enabled: true
-        inputs:
-          mandatory:
-            - Node.fqdn
-            - plugin_name
-            - plugin_conf_params
-            - Service.name
-        post_run:
-          - tendrl.node_agent.objects.Service.atoms.check_service_status
-        pre_run:
-          - tendrl.node_agent.objects.Node.atoms.check_node_up
-        run: tendrl.node_monitoring.flows.configure_collectd.ConfigureCollectd
-        type: Create
-        uuid: dc8fff3a-34d9-4786-9282-55eff6abb6c4
-  Config:
-    time_series:
-      time_series_db: 'graphite'
-      time_series_db_server: '0.0.0.0'
-      time_series_db_port: 80
-    tendrl_performance:
-      performance_server_interface: '0.0.0.0'
-      log_cfg_path = '/etc/tendrl/performance_monitoring_logging.yaml'
-      log_level = 'DEBUG'
+  flows:
+    ConfigureCollectd:
+      atoms:
+        - tendrl.node_agent.objects.Node.atoms.cmd
+      description: "Execute given command on given node"
+      enabled: true
+      inputs:
+        mandatory:
+          - Node.fqdn
+          - plugin_name
+          - plugin_conf_params
+          - Service.name
+      post_run:
+        - tendrl.node_agent.objects.Service.atoms.check_service_status
+      pre_run:
+        - tendrl.node_agent.objects.Node.atoms.check_node_up
+      run: tendrl.node_monitoring.flows.configure_collectd.ConfigureCollectd
+      type: Create
+      uuid: dc8fff3a-34d9-4786-9282-55eff6abb6c4
+  Tendrl_definition: '/_tendrl/definitions/integrations.d/performance_monitoring/data'
```

Comment:
These do not confirm to Tendrl Definitions schema,  Tendrl_definition and Config are "objects"